raiden/network/blockchain_service.py

```diff
@@ -251,8 +249,9 @@ def user_deposit(self, address: Address) -> UserDeposit:
         return self.address_to_user_deposit[address]
 
     def get_address_from_rns(self, address=None) -> str:
-        contract = self.client.new_contract(RNS_RESOLVER_ABI, RNS_RESOLVER_ADDRESS)
-        proxy = ContractProxy(self.client, contract)
-        eip137hash = namehash(address)
-        resolved_address = proxy.contract.functions.addr(eip137hash).call()
+        from rns_sdk.index import RnsPy
```

Comment:
Is this something that we can move up to the imports section? because i really don't like the dynamic import strategy. I'm not sure  about the performance here.